templates/openstack/what-is-openstack.html

```diff
@@ -2,206 +2,593 @@
+      <p>But there are more differences between OpenStack and traditional virtualisation management platforms. Refer to the following table for more information:</p>
+    </div>
+    <div class="col-12">
+      <table>
+        <thead>
+          <tr>
+            <th></th>
+            <th>OpenStack</th>
+            <th>Virtualisation management platforms</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th><b>TCO</b></th>
+            <td>Low</td>
+            <td>High</td>
+          </tr>
+          <tr>
+            <th><b>Resource management methodology</b></th>
+            <td>“Cattle”</td>
+            <td>“Pets”</td>
+          </tr>
+          <tr>
+            <th><b>Resource provisioning mechanism</b></th>
+            <td>Template-based</td>
+            <td>Image-based</td>
+          </tr>
+          <tr>
+            <th><b>Hardware</b></th>
+            <td>Commodity</td>
+            <td>Specialised</td>
+          </tr>
+          <tr>
+            <th><b>Scalability</b></th>
+            <td>Scale out</td>
+            <td>Scale up</td>
+          </tr>
+          <tr>
+            <th><b>Workload</b></th>
+            <td>Stateless</td>
+            <td>Stateful</td>
+          </tr>
+          <tr>
+            <th><b>Open source</b></th>
+            <td>Yes</td>
+            <td>Mostly no</td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+    <p>
+      <a href="https://ubuntu.com/engage/vmware-to-charmed-openstack">Move from VMware to OpenStack&nbsp;&rsaquo;</a>
+    </p>
+  </div>
+</section>
+
+<section class="p-strip">
   <div class="row">
     <div class="col-8">
-      <h2>
-        Software-defined networks and storage
-      </h2>
-      <p>
-        OpenStack includes the ability to provide on-demand networks, IP addresses, firewalls and routers. Basic capabilities for these are built-in and OpenStack can also be integrated with telco-oriented SDN offerings from network equipment providers such as Juniper, Cisco and Nokia. For a business, the ability to define networks through an API enables fast-paced infrastructure automation and cloud-style operations.
-      </p>
+      <h2>How does OpenStack work?</h2>
+      <p>OpenStack is an open source private cloud platform designed to manage distributed compute, network and storage resources in the data centre. In principle, OpenStack aggregates physical resources into one big pool and allocates virtual resources out of this pool to users who can request them on-demand through a self-service portal or application programming interfaces (APIs). But OpenStack itself does not handle virtualisation. Instead, it leverages the existing virtualisation technologies. Therefore, OpenStack is more like a wrapper around traditional virtualisation tools, enabling cloud-native capabilities.</p>
+    </div>
+    <div class="row u-align--center u-hide--small">
```

Comment:
This `row` is a child of another `row` which isn't semantically correct - you need to close the previous `row` div before opening another one. Also a child of this is a `<p>` - it needs to be a `col`